app/models/course/assessment/question/programming.rb

```diff
@@ -1,11 +1,21 @@
 # frozen_string_literal: true
+ActiveRecord::ActsAs::InstanceMethods.class_eval do
+  def dup
```

Comment:
Yah, I think you might want to do the same thing as lesson plan. as in, duplicate the actable first.